tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -1705,14 +1711,15 @@ public static void getStatus(Iterable<String> allPaths, CLDRFile file,
             // String value = file.getSourceLocaleID(path, status);
             MissingStatus missingStatus = VettingViewer.getMissingStatus(file, path, latin);
 
-            switch (missingStatus) {
+            MissingStatus adjustedMissingStatus = missingStatus  == MissingStatus.ALIASED ? aliasRemap : missingStatus;
+
```

Comment:
are there any callers of VettingViewer.getMissingStatus, other than this one, for which the adjustment would be wrong? if so, a brief explanatory comment might help, and in any case, maybe the adjustment should happen in getMissingStatus itself; it could depend on a boolean parameter if needed

there's a general problem that each component tends to have its own slight variation on an idea like "missing status", and the variations lead to inconsistencies that are sometimes unintended ... might there be an opportunity for more unification and encapsulation here?

Reply:
there are multiple callers of VettingViewer.getMissingStatus, I think all within VettingViewer, but they all should give a consistent view of the missing status.

Perhaps getMissingStatus() shouldn't be public if it's only called within this object?

Reply:
it's called by TestUtilities -- otherwise, only 3 callers, all in VettingViewer -- unless I'm leaving out a .jsp

Reply:
DataSection.java includes:
```
        CLDRFile.Status sourceLocaleStatus = new CLDRFile.Status();
        String sourceLocale = ourSrc.getSourceLocaleID(xpath, sourceLocaleStatus);
```
Should that change as well for consistency?

Reply:
1. getMissingStatus is only used internally or in tests. I thought about pushing the special logic to getMissingStatus, but decided it was better to have the lower-level method be simpler.
2. I don't know the relation to DataSection.java; the call you have is for getSourceLocaleID. Do you mean "should it change to use the Extended version"? If that's what you mean, I don't think so.

Reply:
>    ... the relation to DataSection.java ...

It's not clear to me. My concern is whether the status as indicated in the dashboard matches the status as shown in the data section and Info Panel. We can study that further after this is merged.

Reply:
It is used in testing; we could incorporate ICU's technique of marking methods with @internal or even come up with another annotation for methods that should only be public for testing purposes (@testing). But that would take more discussion in the cldr infra meeting.

I will add some better comments to the two methods.